lib/src/sv_onvif.h

```diff
@@ -0,0 +1,41 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2021 Axis Communications AB
```

Comment:
I actually think it should be "2025", but I'm not sure. Let's have a look in the other public repositories.